src/views/ResetPassword.vue

```diff
@@ -58,9 +60,37 @@
   </div>
 </template>
 <script>
+import { config } from "@/configurations";
 
 export default {
   name: "ResetPassword",
+  components: {},
+  data() {
+    return {
+      password1: "",
+      password2: "",
+      submitted: false,
+    };
+  },
+  methods: {
+    handleSubmit() {
+      this.submitted = true;
+      const { password1, password2 } = this;
+      const { dispatch } = this.$store;
+      let updateFields = {};
+      if (password1 && password2) {
+        updateFields["password"] = password1;
+        if (password1.trim() !== password2.trim()) {
```

Comment:
*`!===` (3 equal symbols)

Reply:
in 3 equal symbols giving an error, the error is "Unexpected token =".